seat/src/contract.rs

```diff
@@ -522,9 +444,8 @@ mod tests {
         .unwrap();
         let active_primary_sale: QueryResp = from_binary(&active_primary_sale_query).unwrap();
 
-        match active_primary_sale {
-            QueryResp::ActivePrimarySale(Some(sale)) => assert!(sale.disabled),
-            _ => unreachable!(),
+        if let QueryResp::ActivePrimarySale(Some(_sale)) = active_primary_sale {
+            panic!()
```

Comment:
This can probably be refactored to include a bit more information and an error message